api/v1beta1/cloudstackmachine_webhook.go

```diff
@@ -84,15 +84,13 @@ func (r *CloudStackMachine) ValidateUpdate(old runtime.Object) error {
 	}
 	oldSpec := oldMachine.Spec
 
-	errorList = webhookutil.EnsureBothFieldsAreEqual(r.Spec.Offering.ID, r.Spec.Offering.Name, oldSpec.Offering.ID, oldSpec.Offering.Name, "offering", errorList)
```

Comment:
I think the idea was for cloudstackmachines to be immutable - if a user wants to upgrade, they can create a new one and remove the old one. EKS-A supports these upgrades so users should not be stuck on a particular version or offering.

Reply:
@rejoshed does that sound right?

Reply:
Right, I'll update it to only allow changes to the MachineTemplate